package.json

```diff
@@ -13,11 +13,15 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
```

Comment:
This seems like something that will drastically pump up the bundle size, is it really necessary?  Can it just be done as static individual SVGs?  Or better yet, stick with material icons?

Reply:
Treeshaking mitigates it. Not everything is bundled.

Reply:
still, this seems like it would be more worth it to add the svg files themselves, and let svgr take care of them

Reply:
Sounds like a premature optimization. The package also provides an `ImageDefinition` type with the SVG path and width/height.